src/status_im/contexts/wallet/send/select_asset/view.cljs

```diff
@@ -34,9 +35,15 @@
      {:collectibles         collectibles
       :filtered?            search-performed?
       :on-end-reached       #(rf/dispatch [:wallet/request-collectibles-for-current-viewing-account])
-      :on-collectible-press #(rf/dispatch [:wallet/send-select-collectible
-                                           {:collectible %
-                                            :stack-id    :screen/wallet.select-asset}])}]))
+      :on-collectible-press #(let [collectibles-count (utils/collectibles-count %)]
+                               (if (> collectibles-count 1)
+                                 (rf/dispatch [:wallet/select-collectibles-amount
+                                               {:collectible %
+                                                :stack-id    :screen/wallet.select-asset}])
+                                 (rf/dispatch [:wallet/send-collectibles-amount
+                                               {:collectible %
+                                                :amount      1
+                                                :stack-id    :screen/wallet.select-asset}])))}]))
```

Comment:
Navigation should be handled by the wizard config. Check #19123

Reply:
@briansztamfater, thanks, will check it!